### PR TITLE
AI scale with APIM

### PR DESCRIPTION
**Pull request by diberry**
## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/ai/get-started-app-chat-scaling-with-azure-api-management.md](https://github.com/dotnet/docs/blob/c69ff66dfd55cc7cb2207424e3dbde9665dfdeb4/docs/ai/get-started-app-chat-scaling-with-azure-api-management.md) | [CustomerIntent: As a .NET developer new to Azure OpenAI, I want to scale my Azure OpenAI capacity to avoid rate limit errors with Azure API Management.](https://review.learn.microsoft.com/en-us/dotnet/ai/get-started-app-chat-scaling-with-azure-api-management?branch=pr-en-us-40250) |
| [docs/ai/get-started-app-chat-scaling-with-azure-container-apps.md](https://github.com/dotnet/docs/blob/c69ff66dfd55cc7cb2207424e3dbde9665dfdeb4/docs/ai/get-started-app-chat-scaling-with-azure-container-apps.md) | [Scale Azure OpenAI for .NET chat sample using RAG](https://review.learn.microsoft.com/en-us/dotnet/ai/get-started-app-chat-scaling-with-azure-container-apps?branch=pr-en-us-40250) |

### CODE REVIEW

File: docs/ai/get-started-app-chat-scaling-with-azure-api-management.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+[!INCLUDE [py-apim-cleanup](../intro/includes/scaling-load-balancer-cleanup-azure-api-management.md)]
+
+## Sample code
+
+Samples used in this article include: 
+
+* [.NET chat app with RAG](https://github.com/Azure-Samples/azure-search-openai-demo-csharp)
+* [Load Balancer with Azure API Management](https://github.com/Azure-Samples/openai-apim-lb)
+
+## Next step
```
Comment (gewarren) [inline docs/ai/get-started-app-chat-scaling-with-azure-api-management.md:49]:
```suggestion
## Next steps
```

File: docs/ai/get-started-app-chat-scaling-with-azure-api-management.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+* [View Azure API Management diagnostic data in Azure Monitor](/azure/api-management/api-management-howto-use-azure-monitor#view-diagnostic-data-in-azure-monitor)
+* Use [Azure Load Testing](/azure/load-testing/) to load test your chat app with
```
Comment (gewarren) [inline docs/ai/get-started-app-chat-scaling-with-azure-api-management.md:52]:
```suggestion
* Use [Azure Load Testing](/azure/load-testing/) to load test your chat app.
```

File: docs/ai/get-started-app-chat-scaling-with-azure-api-management.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+## Sample code
+
+Samples used in this article include: 
+
+* [.NET chat app with RAG](https://github.com/Azure-Samples/azure-search-openai-demo-csharp)
+* [Load Balancer with Azure API Management](https://github.com/Azure-Samples/openai-apim-lb)
+
+## Next step
+
+* [View Azure API Management diagnostic data in Azure Monitor](/azure/api-management/api-management-howto-use-azure-monitor#view-diagnostic-data-in-azure-monitor)
```
Comment (gewarren) [inline docs/ai/get-started-app-chat-scaling-with-azure-api-management.md:51]:
```suggestion
* [View Azure API Management diagnostic data in Azure Monitor.](/azure/api-management/api-management-howto-use-azure-monitor#view-diagnostic-data-in-azure-monitor)
```

File: docs/ai/get-started-app-chat-scaling-with-azure-api-management.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+# CustomerIntent: As a .NET developer new to Azure OpenAI, I want to scale my Azure OpenAI capacity to avoid rate limit errors with Azure API Management.
+---
+
+# Scale Azure OpenAI for .NET chat using RAG with Azure API Management
+
+[!INCLUDE [aca-load-balancer-intro](~/azure-dev-docs-pr/articles/intro/includes/scaling-load-balancer-introduction-azure-api-management.md)]
+
+## Prerequisites
+
+* Azure subscription.  [Create one for free](https://azure.microsoft.com/free/ai-services?azure-portal=true)
```
Comment (gewarren) [inline docs/ai/get-started-app-chat-scaling-with-azure-api-management.md:16]:
```suggestion
* An Azure subscription. [Create one for free.](https://azure.microsoft.com/free/ai-services?azure-portal=true) 
```

**Review by gewarren — COMMENTED** [review]
Can you also add to the TOC?

**Review by alexwolfmsft — APPROVED** [review]

**Comment by jmatthiesen** [comment]
Actually, looks links to samples also open in the current window. In general I'd prefer to have external windows open for any content that I want to have alongside the current document as I follow along.

For pages that move on to new topics, like the Next Steps links, those make sense to replace the current content.

**Comment by BillWagner** [comment]
@gewarren 

I think this one is ready now.

**Comment by gewarren** [comment]
@diberry Are you ready to merge this?

**Comment by diberry** [comment]
@jmatthiesen I can't see that the .NET sample supports the required env vars. Can you verify that the .NET chat app support LB with these env vars? OPENAI_HOST and AZURE_OPENAI_CUSTOM_URL

**Comment by BillWagner** [comment]
@diberry 

This fell off the radar. What still needs to happen to get this ready to merge?

**Comment by diberry** [comment]
@BillWagner I'm not sure if the .NET sample is supporting this yet. The subject area is now owned by @alexwolfmsft or @ms-johnalex.

**Comment by ms-johnalex** [comment]
@jmatthiesen Can you comment on this? 

> @jmatthiesen I can't see that the .NET sample supports the required env vars. Can you verify that the .NET chat app support LB with these env vars? OPENAI_HOST and AZURE_OPENAI_CUSTOM_URL

**Comment by gewarren** [comment]
Going to close this PR for now. Please reopen when ready to work on it more.

**Pull request closed without merging**